bot/exts/evergreen/trivia_quiz/_game.py

```diff
@@ -0,0 +1,138 @@
+import asyncio
+import random
+
+import discord
+from discord.ext.commands import Context
+from fuzzywuzzy import fuzz
+
+from bot.bot import Bot
+from bot.exts.evergreen.trivia_quiz import _constants
+from bot.exts.evergreen.trivia_quiz import _helpers
+
+
+class Game:
+    """Class that contains information and functions about Tic Tac Toe game."""
+
+    def __init__(self, bot: Bot, ctx: Context, questions: dict, category: str):
+        self.bot = bot
+        self.ctx = ctx
+        self.questions = questions
+        self.category = category
+
+        # A variable to store the game status: either running or not running.
+        self.over = False
+        # A variable to store the person's ID who started the quiz game in a channel.
+        self.game_owner = ctx.author.id
+        # # A variable to store all player's scores for a the trivia quiz session.
+        self.player_scores = {}
+
+    async def play(self) -> None:
+        """Start and handle game."""
+        # Start game if not running.
+        if not self.over:
+            self.game_owner = self.ctx.author.id
+            start_embed = _helpers.make_start_embed(self.category)
+
+            await self.ctx.send(embed=start_embed)  # send an embed with the rules
+            await asyncio.sleep(1)
+
+        # A variable to store the questions which have been sent to the user during the quiz.
+        done_question = []
+        # A variable to store the number of hints used in a quiz question.
+        hint_no = 0
+        # A variable to store the answer to a quiz question.
+        answer = None
+
+        while not self.over:
+            # Exit quiz if number of questions for a round are already sent.
+            if len(done_question) > _constants.QUESTION_LIMIT and hint_no == 0:
```

Comment:
I think checking for `hint_no` is not required here. What do you think?
```suggestion
            if len(done_question) > _constants.QUESTION_LIMIT:
```